ux.symfony.com/templates/_header.html.twig

```diff
@@ -27,12 +27,12 @@
             </div>
 
             <div class="AppNav_menu">
-                <a href="{{ path('app_turbo') }}" class="AppNav_item">Turbo</a>
```

Comment:
I would not remove it:  2nd most installed UX package and one of the most searched 

https://packagist.org/packages/symfony/ux-turbo/stats